Jmvera255 FAQs and Modules Updates (formerly patch 9) (#49)
OSGConnect/connectbook: training/resources/frequently-asked-questions-faq-.md

**What job scheduler is being used on OSG Connect?**

We use use the task scheduling software called HTCondor to schedule and run jobs.
 
**How do I submit a computing job?**

Jobs are submitted via HTCondor scheduler. Please see our [QuickStart](http://support.opensciencegrid.org/support/solutions/articles/5000633410) guide for more details on submitting and managing jobs.

**How many jobs can I have in the queue?**

The number of jobs that are submitted to the queue by any one user should not exceed 10,000. If you have more jobs than that, we ask that you include the following statement in your submit file: 

`max_idle = 2000`  

## Data Storage and Transfer
   
**What is the best way to process large volume of data?**

Use the Stash data system to stage large volumes of data. Please refer the section [Data Solutions](http://support.opensciencegrid.org/support/solutions/folders/5000262152) for more details. 
 

> **How do I transfer my data to and from OSG Connect?

Reviewer: ```suggestion
**How do I transfer my data to and from OSG Connect?**

```

Author: fixed in commit 4269b61

[title]: - "Frequently Asked Questions"

## Getting Started
   
**Who is eligible to become the user of OSG Connect?**

Any researcher affiliated with a U.S. institution (college, university, national laboratory or research foundation) is eligible to become an OSG Connect user. Researchers outside of the U.S. with affiliations to U.S. groups may be eligible for membership if they are sponsored by a collaborator within the U.S. Researchers outside of the U.S. are asked to first
[contact us](mailto:support@osgconnect.net) directly to discuss membership.

**How do I become an user of OSG Connect?**

Please follow the steps outlined in the [Sign Up process](http://osgconnect.net/signup). 

## Software
   
**What software packages are available?**

> The available software are listed [here](https://support.opensciencegrid.org/support/solutions/articles/12000048518). In general, we support most software that fits the distributed high thoroughput computing model. Additionally, we can add software upon request.

Reviewer: ```suggestion
In general, we support most software that fits the distributed high throughput computing model. Users are encouraged to download and install their own software. For some software, we support distributed software modules listed [here](https://support.opensciencegrid.org/support/solutions/articles/12000048518). Software can be added to the modules upon request. 

```

 
**How do I access a specific software application?**

We have implemented modules within OSG Connect to manage the software that is available to users. Modules allow for easy access to a number of software and version options. Our [Accessing Software using Distributed Environment Modules](https://support.opensciencegrid.org/support/solutions/articles/12000048518-accessing-software-using-distributed-environment-modules) page provides more details on how to use modules in OSG Connect.
 
**Are there any restrictions on installing commercial softwares?**

We only provide software that is freely distributable. At present, we do not have or support most commercial software due to licensing issues. 
 
**Can I request for system wide installation of the open source software useful for my research?**

Yes. Please contact <support@osgconnect.net>.  
   
## Running Jobs
   
**What type of computation is a good match or NOT a good match for OSG Connect?**

> It is to carefully check the computational requirements of your workflow. Please see our ["Is the Open Science Grid for You?"](http://support.opensciencegrid.org/solution/categories/5000131843/folders/5000209523/articles/5000632058-is-high-throughput-computing-for-you-) page for more details on how to determine if your work matches up well with OSG Connect's model.

Reviewer: ```suggestion
It is important to carefully check the computational requirements of your workflow. Please see our ["Is the Open Science Grid for You?"](http://support.opensciencegrid.org/solution/categories/5000131843/folders/5000209523/articles/5000632058-is-high-throughput-computing-for-you-) page for more details on how to determine if your work matches up well with OSG Connect's model.

```

**What job scheduler is being used on OSG Connect?**

We use HTCondor to schedule and run jobs.
 
**How do I submit a computing job?**

Jobs are submitted via HTCondor scheduler. Please see our [QuickStart](http://support.opensciencegrid.org/support/solutions/articles/5000633410-osg-connect-quickstart) guide for more details on submitting and managing jobs.

**How many jobs can I have in the queue?**

The number of jobs that are submitted to the queue by any one user should not exceed 10,000. If you have more jobs than that, we ask that you include the following statement in your submit file: 

> `maxidle 2000`  

Reviewer: ```suggestion
`max_idle  = 2000`  

```

Author: nice catch!
